backend/api/serializers/CreditTrade.py

```diff
@@ -281,6 +281,11 @@ class Meta:
                                       "the transaction."
                 }
             },
+            'date_of_written_agreement': {
+                'error_messages': {
+                    'invalid': "Please enter a valid date in the Agreement Date field: YYYY-MM-DD."
```

Comment:
Since in the UI we're inputting the data in the format of mm/dd/yyyy, does YYYY-MM-DD format makes sense?

Reply:
Thank you, Prashanth, for reviewing my PR. From what I can see, the user inputs the agreement date in the YYYY-MM-DD format. Could you please take a look at the attached screenshots to confirm whether we need to make any changes? I have attached two screenshots: one before entering the date and one after.

Before entering the date:
<img width="1728" alt="Screen Shot 2023-10-31 at 10 31 43 AM" src="https://github.com/bcgov/tfrs/assets/2143103/98f3f2ff-6c73-4f4d-8ded-b7c1100a9ba6">

After entering the date:
<img width="1728" alt="Screen Shot 2023-10-31 at 10 31 50 AM" src="https://github.com/bcgov/tfrs/assets/2143103/a24e822a-b3fe-4263-9aa6-d6a77405b094">

Reply:
Yes, you're right. Since in my system I had chosen the date format to be in US format. it was showing as below
![image](https://github.com/bcgov/tfrs/assets/130073308/16fcdbf2-542a-478e-8378-9a4432a5c1b9)

Thanks,

Reply:
Thanks for the info! We can discuss this in the sprint meeting tomorrow and create a new ticket if needed. I think if the user has the option to change the date format, the error message should adjust accordingly.